options/locale/locale_en-US.ini

```
@@ -2306,7 +2306,7 @@ branch.already_exists = A branch named '%s' already exists.
 branch.delete_head = Delete
 branch.delete = Delete Branch '%s'
 branch.delete_html = Delete Branch
-branch.delete_desc = Deleting a branch is permanent. It <strong>CANNOT</strong> be undone. Continue?
+branch.delete_desc = Deleting a branch is permanent after 'git cleanup task' was run (by default once per day). It CANNOT be undone. Continue?
```

```suggestion
branch.delete_desc = Deleting a branch is permanent. Although the deleted branch may exist for a short time before cleaning up, in most cases it CANNOT be undone. Continue?
```

What do you think about this?

---

Ok for me.